platform/linux-generic/pktio/subsystem.c

```diff
@@ -58,7 +60,9 @@ ODP_SUBSYSTEM_CONSTRUCTOR(pktio_ops)
 #ifdef HAVE_PCAP
 	enable_link_pcap_pktio_ops = 1;
 #endif
+#ifdef ODP_PKTIO_SOCKET
 	enable_link_socket_pktio_ops = 1;
```

Comment:
These are temporary flags, which I was planned to remove after Makefiles update to build all pktio_ops into real individual modules and use --whole-archive or --as-no-needed to link them in.

Reply:
True... and should be removed at that point. This is an intermediary step so we can remove  duplicate pktio_ops subsystem implementation